tutorial/2_conversion_pipeline_for_example_time-resolved_ARPES_data.ipynb

```diff
@@ -470,14 +470,11 @@
    "metadata": {},
```

Comment:
The comment above should remove "Here, one can also define a reference id" since I saw that it's no longer used.

Reply:
Thanks, this has been corrected.